include/deal.II/lac/petsc_vector_base.h

```diff
@@ -327,6 +327,15 @@ namespace PETScWrappers
     VectorBase &
     operator=(const PetscScalar s);
 
+    /**
+     * This method assignes the PETSc Vec to the instance of the class.
+     * This is particularly useful when performing PETSc to Deal.II operations
+     * since it allows to reuse the Deal.II VectorBase and the PETSc Vec
+     * without incurring in memory copies.
```

Comment:
```suggestion
     * This method assigns the PETSc Vec to the instance of the class.
     * This is particularly useful when performing PETSc to deal.II operations
     * since it allows one to reuse the VectorBase and the PETSc Vec
     * without incurring in memory copies.
```
or use the ctor.

Reply:
again, the constructor will create a new deal.II object. here we want to use the same deal.ii object